ppdet/modeling/architecture/yolo.py

```diff
@@ -12,38 +12,41 @@
 class YOLOv3(BaseArch):
     __category__ = 'architecture'
     __inject__ = [
-        'anchor',
         'backbone',
+        'neck',
         'yolo_head',
         'post_process',
     ]
 
-    def __init__(self, anchor, backbone, yolo_head, post_process):
+    def __init__(self,
+                 backbone='DarkNet',
+                 neck='YOLOv3FPN',
+                 yolo_head='YOLOv3Head',
+                 post_process='BBoxPostProcess'):
         super(YOLOv3, self).__init__()
-        self.anchor = anchor
         self.backbone = backbone
+        self.neck = neck
         self.yolo_head = yolo_head
         self.post_process = post_process
 
     def model_arch(self, ):
         # Backbone
-        body_feats = self.backbone(self.inputs)
+        blocks = self.backbone(self.inputs)
```

Comment:
unify the output name between yolo and rcnn? mask rcnn uses body_feats as the output name of backbone and neck

Reply:
Done, thanks!